mitiq/interface/conversions.py

```diff
@@ -20,25 +20,24 @@ class CircuitConversionError(Exception):
     pass
 
 
-register_to_dict: Dict[Tuple[str, str], Callable[[Any], Circuit]]
+from_mitiq_dict: Dict[str, Callable[[Circuit], Any]]
```

Comment:
Since `from_mitiq_dict` is global, should we define it as `FROM_MITIQ_DICT`? I like the new name BTW!.